tests/Serializer/BaseSerializationTest.php

```diff
@@ -1576,6 +1615,7 @@ protected function setUp()
         $this->handlerRegistry->registerSubscribingHandler(new DateHandler());
         $this->handlerRegistry->registerSubscribingHandler(new FormErrorHandler(new IdentityTranslator(new MessageSelector())));
         $this->handlerRegistry->registerSubscribingHandler(new ArrayCollectionHandler());
+        $this->handlerRegistry->registerSubscribingHandler(new IteratorHandler());
```

Comment:
In my opinion it should not in current version. It may be breaking changes if somebody has already implemented similar one. We can add it as a default in the new mayor version.

Reply:
well, currently the 2.0 adoption rate is still low, i would like to add it already. 

Waiting an 3.0 in the next years makes no sense to me..

Reply:
Done.